### PR TITLE
Fix state label collision in common address definition

### PR DESCRIPTION
**Pull request by jbalboni**
Connects to https://github.com/department-of-veterans-affairs/vets.gov-team/issues/2869

There are overlapping state codes between US and Canada and we had switched to a single object to hold all labels, so Nevada turned into Nova Scotia.

screen shot 2017-05-18 at 11 44 09 pm

### CODE REVIEW

File: src/js/common/schemaform/definitions/address.js [hunk 1]
```diff
@@ -66,18 +64,23 @@ export function uiSchema(label = 'Address', useStreet3 = false) {
       const isRequired = addressSchema.required.length > 0;
 
       let stateList;
+      let labelList;
       if (country === 'USA') {
         stateList = usaStates;
+        labelList = usaLabels;
```
Comment (cvalarida) [inline src/js/common/schemaform/definitions/address.js]: Idle curiosity: Why `if` `else if` `else if` instead of a switch?

**Review by cehsu — APPROVED** [review]
LGTM

**Review by cvalarida — APPROVED** [review]
Looks good! Glad to see Nevada dry, hot, and landlocked again. 😄

File: src/js/common/schemaform/definitions/address.js (hunk 1 above)
Comment (jbalboni) [inline src/js/common/schemaform/definitions/address.js]: I dislike switch statements in JS. They're easy to mess up, especially if you're not returning from each branch.

**Comment by iantopper** [comment]
LGTM

**Pull request merged**